packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -100,9 +100,11 @@ export interface IGCStats {
 export const GCNodeType = {
     // Nodes that are for data stores.
     DataStore: "DataStore",
+    // Nodes that are within a data store. For example, DDS nodes.
+    SubDataStore: "SubDataStore",
```

Comment:
Any back-compat considerations with modifying this "enum"?

Reply:
This is entirely used within the container runtime so there should not be any back-compat concerns.